_data/changelogs/component-icon-list.yml

```
@@ -2,6 +2,13 @@ title: Icon list
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Removed deprecated xlink references.
+    summaryAdditional: All xlink:href references have been replaced with href and all xlmns:xlink references have been removed.
+    affectsMarkup: true
```

If this PR is scoped to #2926, we should add `affectsGuidance`. It might _only_ need affects guidance since it is the only true user-facing change from this work. The work in#2926 only changes our image attributes and the icon list / icon guidance.

```suggestion
    affectsGuidance: true;
```

Markup changes were captured in uswds/uswds#6165

---

I think this comment is outdated? This PR should include the changelogs for uswds/uswds#6165 (which should all be `affectsMarkup` or `affectsAssets`, and then #2926 should address all the uswds-site xlink updates and include the `affectsGuidance` changelogs.

---

@amyleadem @mahoneycm Just to confirm, this should be `affectsMarkup` and I have nothing to update here?

---

Leaving this to @mahoneycm to answer (but my guess is no action is needed here)

---

Yep! This is outdated I'll go ahead and resolve 👍